From c30453ca249ba1a49e761c5042ee0da939ea5b23 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Tue, 4 Feb 2020 13:17:18 +0100
Subject: [PATCH] fix broken endian check

__LITTLE_ENDIAN is not a thing, only __LITTLE_ENDIAN__ is. It
also does not make sense to create a third case as there are
only two options.
---
 intern/numaapi/source/build_config.h | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/intern/numaapi/source/build_config.h b/intern/numaapi/source/build_config.h
index b8af51a..9725d8b 100644
--- a/intern/numaapi/source/build_config.h
+++ b/intern/numaapi/source/build_config.h
@@ -311,12 +311,10 @@
 #  define ARCH_CPU_PPC_FAMILY 1
 #  define ARCH_CPU_PPC 1
 #  define ARCH_CPU_64_BITS 1
-#  if defined(__BIG_ENDIAN__)
-#    define ARCH_CPU_BIG_ENDIAN 1
-#  elif defined(__LITTLE_ENDIAN)
+#  if defined(__LITTLE_ENDIAN__)
 #    define ARCH_CPU_LITTLE_ENDIAN 1
 #  else
-#    error Please define your endianness
+#    define ARCH_CPU_BIG_ENDIAN 1
 #  endif
 #elif defined(__s390x__)
 #  define ARCH_CPU_S390_FAMILY 1
-- 
2.24.0

